State.py

```diff
@@ -41,7 +41,7 @@ def won(self):
 
 
     def won_triforce_hunt(self):
-        return self.has('Triforce Piece', self.world.triforce_count)
+        return self.has('Triforce Piece', self.world.triforce_goal)
```

Comment:
```suggestion
        return self.has('Triforce Piece', self.world.triforce_goal_per_world)
```

This should check only the Triforce Pieces in the world itself.